core/src/test/java/apoc/trigger/TriggerNewProceduresTest.java

```diff
@@ -532,4 +567,59 @@ public void testInstallTriggerInSystemDb() {
         }
     }
 
+    @Test
+    public void testEventualConsistency() {
```

Comment:
This test is taking > 3 minutes which is very long. We can discuss it in a separate PR if you agree that it's good to split different work threads into different PRs. 

I'm not sure that we should be testing this 100 times, I would have thought that a single time would suffice. Concurrency conflicts are very hard to detect at best of times, but I don't think we ought to slow the whole CI down because of it. 

What do you think?

Reply:
Changed in the related pr